indigo/distribution.yaml

```diff
@@ -4157,6 +4157,15 @@ repositories:
       url: https://github.com/open-rdc/icart_mini.git
       version: indigo-devel
     status: developed
+  idolink_node:
```

Comment:
actually I have just changed the name of the repo from idolink to idolink_node, but forgot to do it on the release repo.
will rename it and open a new PR